playbooks/tasks/portal-mongo-replicaset-check-majority.yml

```diff
@@ -46,7 +46,9 @@
     # replicaset majority check, it doesn't matter that we stop the rest of
     # services.
     - block:
-        - name: Include checking mongo replicaset majority
+        # Get MongoDB replicaset status details
+
```

Comment:
nit: move newline to above comment.

Reply:
There are 2 conventions that we follow:

1) after `- block:` we always start code or comments without newlines inbetween.

2) Comment + optional newline + task code
  - If the comment is related only the one following task, there is no newline between the comment and the task (comment is sticked to the task).
  - If the comment is related to several tasks that follow, there is a newline between the comment and the tasks that follow.

Example:

```
- block:
  # Get abc
  - name: Get abc
    set_fact:
      abc: "{{ some code to get abc is here }}"

- block:
  # Get something

  - name: Run something preparation command
    command: ...
    register: something_preparation_command_result

  - name: Get something out of the result
    set_fact:
      something: "{{ some code to get something out of something_preparation_command_result }}"
```